File: 8.7.2025/Armstrong.c
#include <stdio.h>

int main() {
    int n,o,r,res;
    printf("Enter a number:\n");
    scanf("%d",&n);
    o=n;
    while(o!=0){
        r = o%10;
        res = res+(r*r*r);
        o = o/10;
    }
    if(res == n){
        printf("It's Armstrong number.");
    } else{
         printf("It's not a Armstrong number.");
    }
}

File: 8.7.2025/Armstrong.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        System.out.println("Enter a number:");
        int n = sc.nextInt();
        int o =n;
        int res = 0;
        int r;
        while(o!=0){
            r = o%10;
            res = res+(r*r*r);
            o = o/10;
        }
        if(res == n){
            System.out.println("It's Armstrong number.");
        } else{
            System.out.println("It's not a Armstrong number.");
        }
    }
}
